lutris/util/steam/config.py

```diff
@@ -141,6 +141,28 @@ def get_entry_case_insensitive(config_dict, path):
         logger.error("Steam config %s is empty: %s", config_filename, ex)
 
 
+def read_library_folders(steam_data_dir):
+    """Read the Steam Library Folders config and return it as an object"""
+    def get_entry_case_insensitive(library_dict, path):
+        for key, _value in library_dict.items():
+            if key.lower() == path[0].lower():
+                if len(path) <= 1:
+                    return library_dict[key]
+                return get_entry_case_insensitive(library_dict[key], path[1:])
```

Comment:
Use value instead of library_dict[key] since you get both the key and value ( == library_dict[key]) by calling .items().

Reply:
Oh my goodness, definitely makes me feel sheepish that I didn't use `value`. Good catch